intel_extension_for_transformers/llm/runtime/graph/models/llama/llama.cpp

```diff
@@ -306,14 +306,9 @@ static bool llama_model_eval_internal(model_context& lctx, const model_token* to
         cur = ne_ffn_silu(ctx0, model.layers[il].ffn[0], model.layers[il].ffn[1], model.layers[il].ffn[2], cur);
       } else {
         struct ne_tensor* tmp = ne_mul_mat(ctx0, model.layers[il].ffn[2], cur);
-
         cur = ne_mul_mat(ctx0, model.layers[il].ffn[0], cur);
-
-        // SILU activation
```

Comment:
no need change llama.cpp right?
but it's all right~

Reply:
delete blank rows only for cleancode